packages/compass-preferences-model/src/preferences-schema.ts

```diff
@@ -56,6 +56,7 @@ export type UserConfigurablePreferences = PermanentFeatureFlags &
     enableAggregationBuilderExtraOptions: boolean;
     enableHackoladeBanner: boolean;
     enablePerformanceAdvisorBanner: boolean;
+    userCanHaveMaximumNumberOfActiveConnections?: number;
```

Comment:
Nit: we don't usually prefix preferences with `userCanHave`, just `maximumNumber...` should be enough here

Reply:
I'll update the TD with the new naming and change it here.